dbdpg/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=dbdpg
+pkg_origin=core
+pkg_version="3.7.4"
+pkg_maintainer="The Habitat Maintainers humans@habitat.sh"
+pkg_license=('Artistic-1.0-Perl' 'GPL-2.0')
+pkg_deps=(
+  core/glibc
+  core/perl
+  core/postgresql
+  core/zlib
+)
+pkg_build_deps=(
+  core/cpanminus
+  core/local-lib
+  core/gcc
+  core/make
+)
+pkg_description="DBD::Pg is a Perl module that works with the DBI module to provide access to PostgreSQL databases."
+pkg_upstream_url="http://search.cpan.org/dist/DBD-Pg/"
+
+do_setup_environment() {
+  push_buildtime_env PERL5LIB "${pkg_prefix}/lib/perl5/x86_64-linux-thread-multi"
+  push_runtime_env PERL5LIB "${pkg_prefix}/lib/perl5/x86_64-linux-thread-multi"
```

Comment:
IIRC, you can just use push_runtime_env and it should be present at build time too (follows the same logic as for pkg_deps).

Reply:
ah neat, I didn't know that!